tools/bazar/actions/BazarListeAction.php

```diff
@@ -179,84 +195,49 @@ public function run()
-            $entries = $entryManager->search(
-                [
-                    'queries' => $this->arguments['query'],
-                    'formsIds' => $this->arguments['idtypeannonce'],
-                    'keywords' => $_REQUEST['q'] ?? '',
-                    'user' => $this->arguments['user'],
-                    'minDate' => $this->arguments['dateMin'],
-                    'correspondance' => $this->arguments['correspondance'] ?? ''
-                ],
-                true // filter on read ACL
-            );
-
-            // call to appendDisplayData removed because already done in EntryManager->search
-        }
-
-        // filter entries on datefilter parameter
-        if (!empty($this->arguments['datefilter'])) {
-            $entries = $this->getService(EntryController::class)->filterEntriesOnDate($entries, $this->arguments['datefilter']) ;
-        }
-
-        // Sort entries
-        if ($this->arguments['random']) {
-            shuffle($entries);
-        } else {
-            usort($entries, $this->buildFieldSorter($this->arguments['ordre'], $this->arguments['champ']));
-        }
-
-        // Limit entries
-        if ($this->arguments['nb'] !== '') {
-            $entries = array_slice($entries, 0, $this->arguments['nb']);
-        }
+            $entries = $bazarListService->getEntries($this->arguments, $forms);     
+            $filters = $bazarListService->formatFilters($this->arguments, $entries, $forms);  
 
-        $filters = $this->formatFilters($entries, $forms);
-
-        $this->arguments['nbbazarliste'] = $GLOBALS['_BAZAR_']['nbbazarliste'] ;
-
-        return $this->render('@bazar/entries/list.twig', [
-            'listId' => $GLOBALS['_BAZAR_']['nbbazarliste'],
-            'filters' => $filters,
-            'renderedEntries' => $this->renderEntries($entries),
-            'numEntries' => count($entries),
-            'params' => $this->arguments,
-            // Search form parameters
-            'keywords' => $_GET['q'] ?? '',
-            'pageTag' => $this->wiki->getPageTag(),
-            'forms' => count($this->arguments['idtypeannonce']) === 0 ? $forms : '',
-            'formId' => $this->arguments['idtypeannonce'][0] ?? null,
-            'facette' => $_GET['facette'] ?? null,
-        ]);
+            // To handle multiple bazarlist in a same page, we need a specific ID per bazarlist
+            // We use a global variable to count the number of bazarliste action run on this page
+            if (!isset($GLOBALS['_BAZAR_']['nbbazarliste'])) {
+                $GLOBALS['_BAZAR_']['nbbazarliste'] = 0;
+            }
+            ++$GLOBALS['_BAZAR_']['nbbazarliste'];
+            $this->arguments['nbbazarliste'] = $GLOBALS['_BAZAR_']['nbbazarliste'] ;
+            
+            // TODO put in all bazar templates
+            $this->wiki->AddJavascriptFile('tools/bazar/libs/bazar.js');
+
+            return $this->render('@bazar/entries/index.twig', [
```

Comment:
```suggestion
            return $this->render('@bazar/entries/list.twig', [
```

je ne suis pas partisan pour appeler ceci `index` .
Le terme `list` précédemment choisi était bien non ?